pkgs/tools/networking/teavpn2/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+stdenv.mkDerivation {
+  pname = "teavpn2";
+  version = "unstable-2023-07-25";
+
+  src = fetchFromGitHub {
+    owner = "TeaInside";
+    repo = "teavpn2";
+    rev = "b21898d001a2e7b821e045162dd18f13561cb04b";
+    hash = "sha256-0/eHK2/+pn6NfawL1xLJv4jDBFvLwELSXNWLUvff1gs=";
+  };
+
+  patches = [ ./nix.patch ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm 0755 teavpn2 $out/bin/teavpn2
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "An open source VPN Software";
+    homepage = "https://github.com/TeaInside/teavpn2";
+    license = licenses.gpl2Plus;
+    mainProgram = "teavpn2";
+    maintainers = with maintainers; [ ludovicopiero ];
+    platforms = [ "x86_64-linux" ];
```

Comment:
Does this only work on `x86_64-linux`?
I mean, it may work on `aarch64-linux` (and `i686-linux`) as well.

Reply:
Oh yeah, the creator of the package just confirmed that it works on any x86 family. So, should I use platforms.linux here?

Reply:
Yes, let's set `platforms.linux` and see the ofborg results.

Reply:
Done